.github/workflows/ruby.yml

```diff
@@ -0,0 +1,27 @@
+name: Ruby
+
+on:
+  push:
+  pull_request:
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version: ['2.6', '2.7', '3.0', '3.1', 'jruby']
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Install redis
+      run: sudo apt-get install -y --no-install-recommends redis-server
```

Comment:
redis-rack using redis-store-testing for testing. redis-store-testing launch `redis-server` command in `redis:test:suite` task.
So I install redis-server in main container without redis service container.